src/main/java/com/mraof/minestuck/client/renderer/entity/GristRenderer.java

```diff
@@ -26,23 +27,45 @@ public GristRenderer(EntityRendererProvider.Context context)
 	@Override
 	public void render(GristEntity grist, float entityYaw, float partialTicks, PoseStack poseStack, MultiBufferSource bufferIn, int packedLightIn)
 	{
+		double shakeScale = 1.0 / 256.0;
```

Comment:
Maybe this could be moved up to line 19, and made into a static final?

Reply:
yurp

Reply:
np[e